apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/DataSourceWrapper.java

```diff
@@ -260,10 +261,10 @@ public void shutdown() {
             stmt.execute("SHUTDOWN COMPACT");
             shutdown = true;
             initialized = false;
+            con.close();
```

Comment:
Looks like con.close may be deleted, because datasource.close will close all connections

Reply:
may be, it's a minor issue.